wfl/Database/MongoDBManager.cs

```diff
@@ -32,5 +32,32 @@ public MongoDBManager()
             IAsyncCursor<RestaurantDto> task = await collection.FindAsync(r => true, null);
             return task.ToEnumerable<RestaurantDto>();
         }
+
+        public async Task<IEnumerable<FoodOrderDto>> GetAllFoodOrdersAsync()
+        {
+            IMongoCollection<FoodOrderDto> collection = db.GetCollection<FoodOrderDto>("food-orders");
+            IAsyncCursor<FoodOrderDto> task = await collection.FindAsync(order => true, null);
+            return task.ToEnumerable<FoodOrderDto>();
+        }
+
+        public async Task InsertFoodOrderAsync(FoodOrderDto order)
+        {
+            order.Id = MongoDB.Bson.ObjectId.GenerateNewId().ToString();
```

Comment:
RestaurantId is being generated the same way currently. The reason why I removed the mongoDB attributes from the Restaurant and FoodOrder classes was to separate the database specific code from our models. I'm not sure if I can have the database generate Id's without placing those attributes back again.